modules/currency-l0/src/main/scala/org/tessellation/currency/l0/cli/method.scala

```diff
@@ -84,7 +86,9 @@ object method {
     prioritySeedlistPath: Option[SeedListPath],
     collateralAmount: Option[Amount],
     globalL0Peer: L0Peer,
-    trustRatingsPath: Option[Path]
+    trustRatingsPath: Option[Path],
+    adminFeeKeyMessage: Option[String],
```

Comment:
These should not be added here, the correct way to implement this is as a new top level command similar to:

```
  object CreateGenesis extends WithOpts[CreateGenesis] {
```

That command is a unique way to create a genesis snapshot, we should implement a new command to produce an admin / staking key message (with a signing function from the wallet.) The options should be removed here, since none of the other commands will ever need to include this value.

Reply:
@ryle-goehausen What is the new command going to be called? `run-genesis-with-signing` or ... ?
And should the new command be a clone of `CreateGenesis` but overriding the two message fields?

Reply:
Hey, sorry, I missed this comment earlier, the goal would be to create a single function, separate from CreateGenesis, that does not implement any of the functionality of create genesis, so it can be used for an existing metagraph.

I.e. the CLI argument should only take those two parameters (with at least one filled,) and use the wallet utilities / signing functions to produce an output blob which we can later submit to the endpoint.